pages/common/git-update-ref.md

```diff
@@ -0,0 +1,12 @@
+# git update-ref
+
+> Git command for creating, updating, and deleting git refs.
+> More information: <https://git-scm.com/docs/git-update-ref>.
+
+- Delete a ref, useful for soft resetting the first commit:
+
+`git update-ref -d {{HEAD}}`
+
+- Update ref with a message:
+
+`git update-ref -m {{"some message"}} {{HEAD}} {{4e95e05}}`
```

Comment:
The quotes should be outside the curly braces. Is the message a commit message?
```suggestion
`git update-ref -m "{{commit message}}" {{HEAD}} {{4e95e05}}`
```
or
```suggestion
`git update-ref -m "{{message}}" {{HEAD}} {{4e95e05}}`
```

Reply:
No, it's not a commit message, but the reason for updating the ref. 

btw after looking at the git commit file  https://raw.githubusercontent.com/tldr-pages/tldr/master/pages/common/git-commit.md, i found that the quote is not added, obviously it is only needed if  there are spaces between the message words, so we can also remove it and only keep message

Reply:
i have removed the quotes, and used the message argugment as it is used in the git commit file

Reply:
If you see the `node` page, we leave the `"` if is required for the message. So please leave the `"`

Reply:
yea but i have removed the spaces, and went with your second suggestion jus ```message``` instead of ```some message```